docs-css/docs-sidebar.css

```diff
@@ -0,0 +1,32 @@
+.sidebar{
+    /* grid-area: "sidebar"; */
+    grid-row-start: 2/3;
+    padding: 2rem 1rem;
+    overflow-y: auto;
+    position: sticky;
+    top: 0rem;
+    height: calc(100vh - 4rem);
+}
+.sidebar h3{
+    padding-left: 1rem;
+    color: var(--text-color1);
+}
+.sidebar-ulist{
+    padding: 2rem 1rem;
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+}
+.sidebar-ulist li a{
```

Comment:
if possible write short meaningful class name

Reply:
Sure will try to name the classes in a better way. Thanks for your suggestion.